client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -92,29 +84,22 @@ export default React.createClass( {
 		}
 
 		return false;
-	},
-
-	getInitialState() {
-		return {
-			actionBarVisible: _actionBarVisible,
-			addPluginTooltip: false
-		};
-	},
+	}
 
 	componentDidMount() {
 		this.debouncedAfterResize = debounce( this.afterResize, 100 );
 		window.addEventListener( 'resize', this.debouncedAfterResize );
-	},
+	}
 
 	componentWillUnmount() {
 		window.removeEventListener( 'resize', this.debouncedAfterResize );
-	},
+	}
 
 	afterResize() {
 		if ( this.props.isBulkManagementActive ) {
 			this.maybeMakeActionBarVisible();
 		}
-	},
+	}
 
 	maybeMakeActionBarVisible() {
```

Comment:
Unfortunately not. Different languages push the button sizes to different sizes and it can kind of look broken. 
So what this does is converts buttons into a drop down down when this is the case.